warehouse/query-core/src/main/java/datawave/query/transformer/ExcerptTransform.java

```diff
@@ -319,51 +337,70 @@ private Set<Excerpt> getExcerpts(PhraseIndexes phraseIndexes) {
+        if (start < 0) {
+            start = 0;
+        }
+        float origHalfSize = (float) (end - start) / 2;
+        int expandSize = 20;
         excerptIteratorOptions.put(TermFrequencyExcerptIterator.FIELD_NAME, field);
-        excerptIteratorOptions.put(TermFrequencyExcerptIterator.START_OFFSET, String.valueOf(start));
-        excerptIteratorOptions.put(TermFrequencyExcerptIterator.END_OFFSET, String.valueOf(end));
-        try {
-            excerptIterator.init(source, excerptIteratorOptions, env);
-            excerptIterator.seek(range, Collections.emptyList(), false);
-            if (excerptIterator.hasTop()) {
-                Key topKey = excerptIterator.getTopKey();
-                String[] parts = topKey.getColumnQualifier().toString().split(Constants.NULL);
-                // The column qualifier is expected to be field\0phrase.
-                if (parts.length == 2) {
-                    return getHitPhrase(hitTermValues, parts);
+        int i = 0;
+        while (i <= 1) {
+            // set the start of the range
+            if (i == 0) {
+                excerptIteratorOptions.put(TermFrequencyExcerptIterator.START_OFFSET, String.valueOf(start));
+            } else {
+                if (start - expandSize > 0) {
+                    excerptIteratorOptions.put(TermFrequencyExcerptIterator.START_OFFSET, String.valueOf(start - expandSize));
                 } else {
-                    log.warn(TermFrequencyExcerptIterator.class.getSimpleName() + " returned top key with incorrectly-formatted column qualifier in key: "
-                                    + topKey + " when scanning for excerpt [" + start + "," + end + "] for field " + field + " within range " + range);
-                    return null;
+                    excerptIteratorOptions.put(TermFrequencyExcerptIterator.START_OFFSET, String.valueOf(0));
                 }
-            } else {
-                return null;
             }
-        } catch (IOException e) {
-            throw new RuntimeException("Failed to scan for excerpt [" + start + "," + end + "] for field " + field + " within range " + range, e);
-        }
-    }
-
-    private String getHitPhrase(ArrayList<String> hitTermValues, String[] phraseParts) {
-        List<String> hitPhrase = new ArrayList<>();
-        for (String phrasePart : phraseParts[1].split(Constants.SPACE)) {
-            if (hitTermValues.contains(phrasePart)) {
-                hitPhrase.add("[" + phrasePart + "]");
+            if (i == 0) {
+                excerptIteratorOptions.put(TermFrequencyExcerptIterator.END_OFFSET, String.valueOf(end));
             } else {
-                hitPhrase.add(phrasePart);
+                // set the end of the range
+                excerptIteratorOptions.put(TermFrequencyExcerptIterator.END_OFFSET, String.valueOf(end + expandSize));
             }
-        }
+            if (log.isDebugEnabled() && i != 0) {
```

Comment:
Whenever we check which attempt we're on, try to be consistent - instead of using `i != 0` in some cases and `i == 1` in others, let's just use `i == 1` for example. 

Specifically talking about the checks in this line (364), and lines 375 and 391